public/app/features/visualization/data-hover/ExemplarHoverView.tsx

```diff
@@ -42,7 +42,21 @@ export const ExemplarHoverView = ({ displayValues, links, header = 'Exemplar' }:
       {links && links.length > 0 && (
         <div className={styles.exemplarFooter}>
           {links.map((link, i) => (
-            <LinkButton key={i} href={link.href} className={styles.linkButton}>
+            <LinkButton
+              key={i}
+              href={link.href}
+              className={styles.linkButton}
+              onClick={
+                link.onClick
+                  ? (event) => {
+                      if (!(event.ctrlKey || event.metaKey || event.shiftKey) && link.onClick) {
+                        event.preventDefault();
+                        link.onClick(event);
+                      }
+                    }
+                  : undefined
+              }
+            >
```

Comment:
```suggestion
<DataLinkButton link={link} key={i} buttonProps={{ size: 'md' }} />
```

Reply:
@javiruiz01 I think we can just reuse `DataLinkButton` and cleanup the file and we should be good to merge!

Reply:
Oh that's so much better, I updated it, thanks!